<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>281</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>281</width>
    <height>281</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>281</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>15 Puzzle C++ by inxane</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn14">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>14</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn15">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: black;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
